Src/ReadGps.c

```diff
@@ -7,14 +7,109 @@
 #include "Data.h"
 
 static int READ_GPS_PERIOD = 1000;
+static int CMD_TIMEOUT = 1500;
+
+static const uint8_t GPS_SET_ANTENNA[] = {0xB5, 0x62, 0x06, 0x13, 0x04, 0x00, 0x1B, 0x00, 0x8B, 0xA9}; // CFG-ANT
```

Comment:
Where is B5 from? I dont see it in the datasheet.

Reply:
it's the header value. That and 0x62

Reply:
What page of the datasheet are you getting this from?